include/smack/SmackRep.h

```diff
@@ -184,8 +186,10 @@ class SmackRep {
   virtual vector<string> getModifies();
   unsigned numElements(const llvm::Constant* v);
   void addInit(unsigned region, const llvm::Value* addr, const llvm::Constant* val);
+  void addInitFunc(const Stmt* s);
```

Comment:
These are actually function calls, not function declaration/definitions.  Does an llvm::Function refer to a function call?

The backing data structure for this (vector<Stmt*> initFuncs) contains a list of call statements that get injected into the beginning of main().

Am I missing something here?